tests/translate/from_native_test.py

```diff
@@ -199,3 +199,14 @@ def test_eager_only_lazy_dask(eager_only: Any, context: Any) -> None:
     with context:
         res = nw.from_native(dframe, eager_only=eager_only)
         assert isinstance(res, nw.LazyFrame)
+
+
+def test_from_native_strict_false_typing() -> None:
+    df = pl.DataFrame()
+    nw.from_native(df, strict=False)
+    nw.from_native(df, strict=False, eager_only=True)
+    nw.from_native(df, strict=False, eager_or_interchange_only=True)
+
+    unstable_nw.from_native(df, strict=False)
+    unstable_nw.from_native(df, strict=False, eager_only=True)
+    unstable_nw.from_native(df, strict=False, eager_or_interchange_only=True)
```

Comment:
question out of curiosity: it's not super clear to me what we are testing here. wouldn't `mypy` be able to catch errors for typing? 👀

Reply:
yeah it's just to check that mypy doesn't complain on these lines

Reply:
In our previous codebase, there was no such combination of parameters present, neither in 'narwhals' nor in 'tests'.

Thus, I believe this is the reason why the typing issue is resolved now, rather than before.